clang/lib/AST/MicrosoftMangle.cpp

```diff
@@ -2408,6 +2408,9 @@ void MicrosoftCXXNameMangler::mangleType(const BuiltinType *T, Qualifiers,
   //                 ::= _U # char32_t
   //                 ::= _W # wchar_t
   //                 ::= _Z # __float80 (Digital Mars)
+  //                 ::= FIXME # _Decimal32
+  //                 ::= FIXME # _Decimal64
+  //                 ::= FIXME # _Decimal128
```

Comment:
We won't know what to put here until Microsoft ships this feature or releases information on their planned mangling for these types (assuming an intent to support them in C++).

Reply:
I filed issue #18 to follow up on this.